build.gradle

```diff
@@ -41,6 +41,7 @@ task coverage(type: JacocoReport) {
 }
 
 dependencies {
+    testImplementation 'junit:junit:4.13.1'
```

Comment:
Hi. May I know why this line is added?

Reply:
There was something that could ignore test cases, but tbh it didn't work well. I think I'll remove this.